00_ES6/00_start/src/pages/chat/chat.container.business.js

```diff
@@ -0,0 +1,29 @@
+import { createSocket } from '../../api/chat'
+
+export const establishRoomSocketConnection = (nickname, room) => {
+    // TODO: move this to env variable
+    const baseUrl = 'http://localhost:3000';
+
+    const socketParams = {
+      url: baseUrl,
+      channel: room,
+      options: {
+        query: `user=${nickname}`
+      },
+    };
+    
+    return createSocket(socketParams);        
+}
+
+export const messagesToString = (messages) => {
+  let content = '';
+  
+  messages.map((ms) => ({
```

Comment:
Use reduce

Reply:
This structure will change, going to merge it and wait for new structure to merge, thanks for the tip